Bootstrap/Update.php

```diff
@@ -627,4 +628,20 @@ private function addOverwriteMainImage()
             }
         }
     }
+
+    private function changeGroupNameImportSettings()
+    {
+        if (version_compare($this->version, '1.1.12', '<=')) {
```

Comment:
`$this->version`contains the old version so when updating from 1.1.11 to 1.1.12 it would contain 1.1.11 
i think you have to check less or eguals '1.1.11'
in general it is Version_with_that_change - 1